inst/rmarkdown/templates/meningitis_outbreak/skeleton/skeleton.Rmd

```diff
@@ -990,64 +1118,101 @@ tab_linelist(linelist_cleaned,
   kable(digits = 2)
 ```
 
-`r fmt_count(linelist_cleaned, previously_vaccinated == "Yes")` reported
-previous vaccination. `r fmt_count(linelist_cleaned, previously_vaccinated == "No")` were unvaccinated and `r fmt_count(linelist_cleaned, previously_vaccinated == "Unknown")` were unsure.
 
 
-Cases by symptoms. 
 
+
+Of the patients, `r fmt_count(linelist_cleaned, patient_facility_type == "Outpatient")` were seen as outpatients and `r fmt_count(linelist_cleaned, patient_facility_type == "Inpatient")` were inpatients. Among inpatients, the median number of days admitted was `r median(linelist_cleaned$obs_days, na.rm = T)`, 
+with a range between `r min(linelist_cleaned$obs_days, na.rm = T)` and `r max(linelist_cleaned$obs_days, na.rm = T)` days.
+
+
+
+Cases by symptoms
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/// describe_by_symptoms \\\
+--------------------------------------------------------------------------------
+
+This chunk gives the counts and proportions for all of the variables in SYMPTOMS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
 ```{r describe_by_symptoms}
-# get counts and proportions for all variables named in SYMPTOMS
+
 tab_linelist(linelist_cleaned, SYMPTOMS, keep = "Yes") %>% 
-  # re-order columns (drop total_prop too)
   select(-value) %>%
-  # fix the way symptom names are displayed - remove the _ and make the first letter uppercase
+  # fix the way symptom names are displayed
+  # gsub switches underscore for space in the variable column
+  # str_to_sentence makes the first letter capital, and all others lowercase
   mutate(variable = str_to_sentence(gsub("_", " ", variable))) %>%
   # rename accordingly
-  rename("Symptom"   = variable,
-         "Cases (n)" = n,
-         "%"         = proportion) %>%
+  rename_redundant("%" = proportion) %>%
+  augment_redundant(" (n)" = " n$") %>%
   kable(digits = 2)
-
 ```
 
-If available, you can also describe the cases by lab results 
 
+
+Cases by lab results 
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+/// describe_by_labs \\\
+--------------------------------------------------------------------------------
+
+
+This chunk gives the counts and proportions for all of the variables in LABS.
+If you do not have lab results, comment out this chunk.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
 ```{r describe_by_labs}
 
-# get counts and proportions for all variables named in LABS
 tab_linelist(linelist_cleaned, LABS, 
-             transpose = "value") %>% 
-  # fix the way lab test names are displayed
-  mutate(variable = str_to_sentence(gsub("_", " ", variable))) %>%
+             transpose = "variable", col_total = TRUE) %>%
```

Comment:
@zkamvar in tab_linelist if you transpose ="variable", then col_total is no longer on the bottom row

Reply:
Zhian's notes:

The reason is because the different LABS variables all have Positive, Negative, but malaria rdt has "Not Done" while the others have "Unknown". When you stack the results on top of one another, then you get the problem where you have "Total" appearing before "Not Done", and that screws up the factor levels :man_facepalming: 

What I need to do is to combine the factor levels of the incoming vectors (assuming "drop/keep" hasn't been called) and apply those with "Total" to the factor levels at the end.

Reply:
ah nice - maybe there should just be a cleaning step that merges unknown + not done in the template too

Reply:
should be fixed in 5f0ebe8